third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -5197,12 +5209,15 @@ void OnNewWindow1(v8::Isolate* isolate, LocalFrame* localFrame) {
                       ReadFromRecordingDirectory);
   SetFunctionProperty(isolate, args, "getRecordingFilePath",
                       GetRecordingFilePath);
-  SetFunctionProperty(isolate, args, "getPersistentId", GetPersistentId);
-  SetFunctionProperty(isolate, args, "checkPersistentId", CheckPersistentId);
+  SetFunctionProperty(isolate, args, "getPersistentId", fromJsGetPersistentId);
+  SetFunctionProperty(isolate, args, "checkRecordReplayObjectId", CheckRecordReplayObjectId);
```

Comment:
Why rename this?  A lot of changes in this PR are confusing to me.  This is a debugging function and while I don't believe it's currently used anywhere, renaming things like this can break things for no reason.

Reply:
Good point. I'll change `checkPersistentId` to be in line with `getPersistentId` then, then there is no renaming necessary.

Reply:
It feels like this has always been a temporary utility function, a fancy version of a `Print` statement 🤷